src/main/java/com/amazon/jenkins/ec2fleet/CloudNanny.java

```diff
@@ -70,6 +72,34 @@ private static List<Cloud> getClouds() {
         return Jenkins.get().clouds;
     }
 
+    private void updateCloudWithScaler(EC2FleetCloud oldCloud) throws IOException {
```

Comment:
I think we would still want to add a few simple unit tests for these branches even though it is a temporary change for backwards compatibility.

Reply:
The functionality for it gets tested in all of the existing tests. I added in one new test that specifically checks that the new Executor Scalers are updated properly.